Library/Core/UnoCore/Source/Uno/Bytes.uno

```diff
@@ -0,0 +1,721 @@
+using Uno.Compiler.ExportTargetInterop;
+
+namespace Uno
+{
+    public static class Bytes
```

Comment:
Hmm, I think `Uno.Bytes` is a bit too abstract of a name... could we use something like `Uno.ByteArrayExtensions` or something like that instead? `Bytes` sounds like something we *might* want to keep for something else in the future...

Reply:
OK - better safe than sorry, and since these are extension methods the actual name of the class won't be used much. Updated to `ByteArrayExtensions` now. 👍